src/utils/dom.js

```diff
@@ -15,6 +15,12 @@ export const states = {
  * Add modal + overlay to DOM
  */
 export const init = (params) => {
+  // Clean up old modals
+  while (getContainer()) {
```

Comment:
There could be only one modal and one container at the time, this loop isn't needed I think.